examples/factors/src/main.rs

```diff
@@ -12,15 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use factors_methods::{MULTIPLY_ELF, MULTIPLY_ID};
+use factors::multiply_factors;
+use factors_methods::MULTIPLY_ID;
 use risc0_zkvm::{
     serde::{from_slice, to_vec},
     Executor, ExecutorEnv, SessionReceipt,
 };
 
+#[cfg(doctest)]
+mod test_readme {
+    macro_rules! external_doc_test {
+        ($x:expr) => {
+            #[doc = $x]
+            extern "C" {}
+        };
+    }
+
+    external_doc_test!(include_str!("../README.md"));
+}
```

Comment:
Why do we need this *and* have lib.rs refer to README.md to get test coverage?

Reply:
Unnecessary, will remove. Artifact of re-testing after refactor whether we really needed the lib